client/app/expenses/hooks/useCreateExpense.ts

```diff
@@ -0,0 +1,55 @@
+import { useForm } from 'react-hook-form'
+import { useGetHouseInfo, useToast } from '@/_hooks'
+import { isErrorWithMessage } from '@/_utils'
+import { useCreateExpenseMutation } from './api'
+import { CreateExpenseSchema, createExpenseResolver } from '../schema'
+
+export const useCreateExpense = () => {
+  const { toast } = useToast()
+  const houseId = useGetHouseInfo().houseId
+
+  const { mutate, isPending } = useCreateExpenseMutation()
+
+  const form = useForm<CreateExpenseSchema>({
+    resolver: createExpenseResolver,
+    // TODO: Set empty to fee if I figured out a way
+    defaultValues: {
+      itemName: '',
+      fee: 0,
```

Comment:
It's hard to set null as default value bcause of type, so I'll fix it when I have time in the future

Reply:
Okay! I'll fix it in the future PR!